howtos/rpm_install.rst

```diff
@@ -0,0 +1,65 @@
+.. _rpm_install:
+
+Install on Vanilla openSUSE/SuSE SLES
+=====================================
+
+Our installer profiles (`<https://github.com/rockstor/rockstor-installer>`_), and consequently our pre-built installers, deviate only a little from openSUSE’s JeOS appliance profiles.
+As a result, it is possible to manually configure a Vanilla openSUSE/SuSE SLES based install to then install Rockstor's rpm.
+
+.. warning::
+
+    **For advanced users only**
+
+    This install method is predominantly for developers or advanced users and will likely require additional maintenance.
+    It may also serve as an option where a SLES base is required. Please note our undesirable disabling of apparmor.
+    We hope to resolve this partial failure in due time.
+
+The procedure, not recommended or supported, consists in adding the relevant Rockstor repositories to a dedicated vanilla openSUSE / SuSE SLES non transactional server install.
```

Comment:
```suggestion
The procedure, not recommended or supported, consists of adding the relevant Rockstor repositories to a dedicated vanilla openSUSE / SuSE SLES non transactional server install.
```

Reply:
Trivial change likely not worth holding up the pr merge and publish I think. But noting for later reference.